### ISSUE TITLE
MailScanner not controlling/starting sendmail in 5.0.3 on CentOS/RHEL7?

### ISSUE THREAD

**Issue by sadsfae** [MailScanner/v5#50]
Hi Folks,

On CentOS 7 I seem to be having issues with the stock ```5.0.3``` MailScanner starting sendmail like it used to in RHEL5.x or RHEL6.x.  

I am currently running sendmail via systemd but I'm not sure that MailScanner is really fully working (though it does start).  I see only partial messages in logs even though I've set logging to show everything.  Having used MailScanner with CentOS5.x I recall that it started and controlled sendmail.  

```
Apr 24 19:06:26 funcamp MailScanner[9661]: MailScanner Email Processor version 5.0.3 starting...
Apr 24 19:06:26 funcamp MailScanner[9661]: Reading configuration file /etc/MailScanner/MailScanner.conf
Apr 24 19:06:26 funcamp MailScanner[9661]: Read 1501 hostnames from the phishing whitelist
Apr 24 19:06:26 funcamp MailScanner[9661]: Read 14367 hostnames from the phishing blacklists
Apr 24 19:06:26 funcamp MailScanner[9661]: Using SpamAssassin results cache
Apr 24 19:06:26 funcamp MailScanner[9661]: Connected to SpamAssassin cache database
Apr 24 19:06:26 funcamp MailScanner[9661]: Enabling SpamAssassin auto-whitelist functionality...
Apr 24 19:06:27 funcamp MailScanner[9661]: Auto: Found virus scanners: clamav
Apr 24 19:06:27 funcamp MailScanner[9661]: Connected to Processing Attempts Database
Apr 24 19:06:27 funcamp MailScanner[9661]: Found 0 messages in the Processing Attempts Database
Apr 24 19:06:27 funcamp MailScanner[9661]: Using locktype = posix
```

Starting/running sendmail via systemd is the only way to receive mail, and I would expect to see some more logging with these options enabled

When I receive mail I only see the normal sendmail messages and do not see anything MailScanner related despite the following logging options.

```
Syslog Facility = mail
Log Spam = yes
Log Non Spam = yes
Log SpamAssassin Rule Actions = yes
```

Is this related to issue #23 ?

**Comment by shawniverson** [comment]
Work in progress here https://github.com/MailScanner/v5/pull/54

**Comment by shawniverson** [comment]
Resolved in v5.0.6-1
